thegreenweb/background.js

```diff
@@ -166,4 +166,8 @@ function showIcon(resp,tabId)
     chrome.pageAction.setIcon({'tabId' : tabId, 'path' : icon});
```

Comment:
should we replace chrome here with browser now? (new ticket)

Reply:
Good catch. I think it _might_ still work, as the the chrome object is supported by both, but doesn't return Promises for certain functions.

The code in background.js doesn't use any promises so we're probably safe, but creating a ticket to harmonise on the use of the browser object is a good idea, and would make a good first issue.

The browser object is a proposed name in for this object in the the cross-browser standard being worked out:

https://browserext.github.io/browserext/

Reply:
i remember i've kept it all on chrome to make it clear this extension started on chrome and that firefox was supposed to support it. With the polyfill we can clean that up a bit already.